src/Microsoft.VisualStudio.Web.CodeGeneration.Tools/project.json

```diff
@@ -27,11 +27,11 @@
       "version": "1.1.0-*",
       "type": "build"
     },
-    "Microsoft.VisualStudio.Web.CodeGeneration": "1.0.0-*"
+    "Microsoft.VisualStudio.Web.CodeGeneration": "1.1.0-*"
   },
   "frameworks": {
     "net451": {},
-    "netcoreapp1.0": {
```

Comment:
I believe this still needs to be netcoreapp1.0 because CLI doesn't recognize tools in the netcoreapp1.1 TFM